lib/bitstyles_phoenix/components/icon.ex

```diff
@@ -11,26 +11,48 @@ defmodule BitstylesPhoenix.Components.Icon do
 
   `opts[:name]` — The name of the icon to render. This must be present in the SVG that renders all the icons in the top of the body e.g. `_svgs.html.eex` as a `<symbol>` containing one filled path.
 
+  `opts[:size]` - Specify the icon size to use. Available sizes are specified in CSS, and default to `s`, `m`, `l`, `xl`. If you do not specify a size, the icon will fit into a `1em` square.
+
   ## Examples
 
       iex> safe_to_string ui_icon("right")
-      ~s(<svg alt="" class="a-icon" role="presentation"><use xlink:href="#icon-right"></svg>)
+      ~s(<svg aria-hidden="true" class="a-icon" focusable="false" height="16" width="16" xmlns="http://www.w3.org/2000/svg"><use xlink:href="#icon-right"></svg>)
 
       iex> safe_to_string ui_icon("right", size: "s")
-      ~s(<svg alt="" class="a-icon a-icon--s" role="presentation"><use xlink:href="#icon-right"></svg>)
+      ~s(<svg aria-hidden="true" class="a-icon a-icon--s" focusable="false" height="16" width="16" xmlns="http://www.w3.org/2000/svg"><use xlink:href="#icon-right"></svg>)
 
       iex> safe_to_string ui_icon("trashcan")
-      ~s(<svg alt="" class="a-icon" role="presentation"><use xlink:href="#icon-trashcan"></svg>)
+      ~s(<svg aria-hidden="true" class="a-icon" focusable="false" height="16" width="16" xmlns="http://www.w3.org/2000/svg"><use xlink:href="#icon-trashcan"></svg>)
+
+      iex> safe_to_string ui_icon("trashcan", width: 20)
+      ~s(<svg aria-hidden="true" class="a-icon" focusable="false" height="20" width="20" xmlns="http://www.w3.org/2000/svg"><use xlink:href="#icon-trashcan"></svg>)
 
       iex> safe_to_string ui_icon("trashcan", class: "foo bar")
-      ~s(<svg alt="" class="a-icon foo bar" role="presentation"><use xlink:href="#icon-trashcan"></svg>)
+      ~s(<svg aria-hidden="true" class="a-icon foo bar" focusable="false" height="16" width="16" xmlns="http://www.w3.org/2000/svg"><use xlink:href="#icon-trashcan"></svg>)
```

Comment:
I guess we should start writing a changelog for this project too :)

Reply:
Everyone gets a changelog! :D I’ll add a ticket

Reply:
https://github.com/bitcrowd/bitstyles_phoenix/issues/21